### PR TITLE
Broadcast creation wizard

### PR DESCRIPTION
**Pull request by ericnewcomer**
This doesn't _quite_ fit with the current broadcast forms, but maybe we do this as the first iteration before changing targeting, etc..

Screenshot 2023-06-05 at 8 21 48 PM
Screenshot 2023-06-05 at 8 07 31 PM
Screenshot 2023-06-05 at 8 12 26 PM

### CODE REVIEW

File: templates/smartmin/delete_confirm.haml [hunk 1]
```diff
@@ -6,7 +6,6 @@
 -block modal-extra-style
   {{block.super}}
   %link{rel:"stylesheet", href:"{{ STATIC_URL }}css/tailwind.css", type:"text/css"}
-  %link{type:'text/css', rel:'stylesheet', href:"{{ STATIC_URL }}fonts/temba/style.css", media:'all'}
```
Comment (ericnewcomer) [inline templates/smartmin/delete_confirm.haml:9]: This file no longer exists

File: temba/msgs/views.py [hunk 2]
```diff
@@ -182,7 +276,7 @@ def build_content_menu(self, menu):
                 menu.add_modax(
                     _("New Broadcast"),
                     "new-scheduled",
-                    reverse("msgs.broadcast_scheduled_create"),
+                    reverse("msgs.broadcast_create"),
```
Comment (ericnewcomer) [inline temba/msgs/views.py:279]: Haven't removed the old creation view, will in later PR

File: temba/msgs/tests.py [hunk 3]
```diff
@@ -1934,6 +1935,28 @@ def test_get_translation(self):
         self.assertEqual(f'<Broadcast: id={broadcast.id} text="Hola a todos">', repr(broadcast))
 
 
+def get_form_data(
```
Comment (rowanseymour) [inline temba/msgs/tests.py:1938]: name could be a bit more specific or nest it where it's used

File: temba/msgs/views.py [hunk 4]
```diff
@@ -168,6 +222,46 @@ class BroadcastCRUDL(SmartCRUDL):
     )
     model = Broadcast
 
+    class Create(OrgPermsMixin, SmartWizardView):
+        form_list = [("compose", ComposeForm), ("target", TargetForm), ("schedule", ScheduleForm)]
+        success_url = "@msgs.broadcast_scheduled"
+        submit_button_name = _("Create Broadcast")
+
+        def get_form_kwargs(self, step):
+            if step in ("schedule", "target"):
```
Comment (rowanseymour) [inline temba/msgs/views.py:231]: would be marginally simpler to just make 'em all take org

**Review by rowanseymour — APPROVED** [review]

File: templates/utils/forms/wizard.html [hunk 5]
```diff
@@ -0,0 +1,45 @@
+{% load i18n smartmin %}
```
Comment (rowanseymour) [inline templates/utils/forms/wizard.html:1]: have added the djlint settings from the hamless branch if you want to update and check this formatting matches that

File: templates/msgs/broadcast_create_schedule.html [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    }
+  </style>
+{% endblock extra-style %}
+{% block extra-script %}
+  <script>
+    var modalBody = getModax("#shared-modax").shadowRoot;
+    var repeatPeriod = modalBody.querySelector("#id_schedule-repeat_period");
+    var repeatDays = modalBody.querySelector(".repeat-days-container");
+
+    console.log(repeatPeriod);
```
Comment (rowanseymour) [inline templates/msgs/broadcast_create_schedule.html:17]: crud

**Review by rowanseymour — CHANGES_REQUESTED** [review]

**Pull request merged**